server/src/test/java/org/candlepin/policy/AutobindRulesTest.java

```diff
@@ -2205,7 +2209,7 @@ public void testSelectBestPoolsShouldSelectBothPoolsInStackWhenOneOfThemProvides
         Pool MCT80 = TestUtil.createPool(owner, prodMCT80);
         MCT80.setId("MCT80");
         MCT80.setQuantity(1L);
-        MCT80.addProvidedProduct(product69);
+        MCT80.getProduct().addProvidedProduct(product69);
```

Comment:
Add the provided product to the product itself up on line 2208.

Reply:
Updated.